packages/SwingSet/misc-tools/elapsed-per-method.py

```diff
@@ -35,26 +37,36 @@
                 msg = vd[2]
                 methargsCD = msg["methargs"]
                 result = msg["result"]
-                method = json.loads(methargsCD["body"])[0]
+                methodobj = json.loads(methargsCD["body"])[0]
+                if isinstance(methodobj, str):
+                    method = methodobj
+                elif isinstance(methodobj, dict) and methodobj["@qclass"] == "symbol":
+                    method = methodobj["name"]
+                else:
+                    raise ValueError("cannot determine method of %s" % json.dumps(methodobj))
```

Comment:
```suggestion
                method = unmarshal(methargsCD["body"])[0]
                if isinstance(methodobj, dict):
                    # assume a capdata symbol and upgrade to the equivalent smallcaps
                    # https://github.com/endojs/endo/blob/master/packages/marshal/src/encodeToSmallcaps.js
                    assert method["@qclass"] == "symbol"
                    method = "%" + methodobj["name"] # probably now %@@asyncIterator
```
which is also a good candidate for abstraction:
```suggestion
                method = unmarshalMethod(methargsCD)
```